config/crd/bases/cloud.redhat.com_clowdjobinvocations.yaml

```diff
@@ -57,6 +57,14 @@ spec:
                     description: Iqe is the job spec to override defaults from the
                       ClowdApp's definition of the job
                     properties:
+                      IbutsuSource:
```

Comment:
When you update the above properties, just make sure to `make pre-push` and the property names here should get updated.